app/components/container/containerController.js

```diff
@@ -116,9 +116,34 @@ function ($scope, $state, $stateParams, $filter, Container, ContainerCommit, Ima
     });
   };
 
-  $scope.remove = function () {
+  $scope.confirmRemove = function () {
+    var force = false;
+    if ($scope.container.State.Running) {
+      force = true;
+      ModalService.confirm({
+        title: "Are you sure?",
+        message: "You are removing a running container. Please confirm your action",
+        buttons: {
+          confirm: {
+            label: 'Force remove the container'
+          },
+          cancel: {
+            label: 'Cancel'
+          }
+        },
+        callback: function (confirmed) {
+          if(!confirmed) { return; }
+          $scope.remove(force);
+        }
+      });
+    } else {
+      $scope.remove(force);
+    }
+  };
+
+  $scope.remove = function(force) {
```

Comment:
Force parameter in the remove function should be optional, see my comment below regarding the value of the force parameter.